src/gui/models/addForcefieldDialogModel.h

```diff
@@ -116,6 +116,7 @@ class AddForcefieldDialogModel : public QObject
     Radio atomTypeRadio_ = Radio::All;
     // The choice for how to handle intramolecular terms
     Radio intramolecularRadio_ = Radio::All;
+    void assignIntramolecularTerms(const Forcefield *ff);
```

Comment:
Just being picky about separating functions from variables.
```suggestion

    private:
    // Assign intramolecular terms to species
    void assignIntramolecularTerms(const Forcefield *ff);
```